### PR TITLE
Add interactive docker-compose for SIO

### PR DESCRIPTION
**Pull request by w3sip**

### CODE REVIEW

**Review by paroque28 — APPROVED** [review]

**Comment by atharvanan1** [comment]
At current state, we can't run this from ui

**Comment by w3sip** [comment]
@atharvanan1 - intended for shell/troubleshooting mode.

**Comment by w3sip** [comment]
@atharvanan1 @paroque28 - also added a trivial script to start the shell

**Comment by atharvanan1** [comment]
Cool!

File: sio/sio-shell.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+docker-compose -f docker-compose.interactive.yml run --rm sio
```
Comment (atharvanan1) [inline sio/sio-shell.sh:3]: Probably helpful to add

```
cd `dirname $0`
```
at the top

**Pull request merged**